src/jpg2pdf.Console/Options.cs

```diff
@@ -0,0 +1,13 @@
+﻿using CommandLine;
+
+namespace jpg2pdf.Console
+{
+	sealed class Options
+	{
+		[Value(0, Required = true, HelpText = "Input files to be processed.")]
+		public IEnumerable<string> InputFiles { get; set; }
+
+		[Option('o', "output", Required = false, HelpText = "Output file.")]
+		public string OutputFile { get; set; }
```

Comment:
Need a option where the program produces the file in the same directory of the input but with the custom filename